### PR TITLE
Package Version support for Meshery Extensions

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: manav1403 <manavavni@gmail.com>

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by sayantan1413 — COMMENTED** [review]
I guess this will be solving the plugin.open() problem. I tried this change and it was working fine for me. So I think this can be merged and then check if the error still persists.

There is a lint failure due to a formatting issue.

**Comment by sayantan1413** [comment]
@Abhishek-kumar09 The UI test failure is due to any Cypress issue? This change doesn't seem to be failing those UI tests right?

**Comment by leecalcote** [comment]
@sayantan1413 correct. I'll rerun the checks just in case....

File: models/remote_provider.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func (l *RemoteProvider) loadCapabilities(token string) {
 	}
 
 	req, _ := http.NewRequest(http.MethodGet, remoteProviderURL.String(), nil)
-
+	// logrus.Debug(remoteProviderURL.String())
```
Comment (Abhishek-kumar09) [inline models/remote_provider.go:98]: remove this line.

File: models/remote_provider.go [hunk 2]
```diff
@@ -124,6 +124,7 @@ func (l *RemoteProvider) loadCapabilities(token string) {
 	if err := decoder.Decode(&l.ProviderProperties); err != nil {
 		logrus.Errorf("[Initialize]: Failed to decode provider properties %s", err)
 	}
+	l.ProviderProperties.PackageVersion = version;
```
Comment (Abhishek-kumar09) [inline models/remote_provider.go:127]: remove semicolon.
or run `gofmt -s -w .`

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: models/remote_provider.go (hunk 1 above)
Comment (leecalcote) [reply models/remote_provider.go:98]: @Abhishek-kumar09 will you commit these corrections directly to this branch? You can use CTL+G on Github's web ui to do so.

File: models/remote_provider.go (hunk 2 above)
Comment (Abhishek-kumar09) [inline models/remote_provider.go:127]:
```suggestion
	l.ProviderProperties.PackageVersion = version
```

File: models/remote_provider.go (hunk 1 above)
Comment (Abhishek-kumar09) [inline models/remote_provider.go:98]:
```suggestion
```

**Comment by leecalcote** [comment]
Let's close this out. Please reopen if needed.

**Pull request closed without merging**